src/teleop/gui/src/components/AutonRoverMap.vue

```diff
@@ -0,0 +1,349 @@
+      <div class="online">
+        <label><input type="checkbox" v-model="online" />Online</label>
+      </div> 
+    </div>
+  </div>
+</template>
+
+<script>
+import { LMap, LTileLayer, LMarker, LPolyline, LPopup, LTooltip, LControlScale } from 'vue2-leaflet'
+import { mapGetters, mapMutations } from 'vuex'
+import L from '../leaflet-rotatedmarker'
+
+const MAX_ODOM_COUNT = 1000
+const DRAW_FREQUENCY = 10
+// Options for the tilelayer object on the map
+const onlineUrl = 'http://{s}.google.com/vt/lyrs=s&x={x}&y={y}&z={z}'
+const offlineUrl = '/static/map/{z}/{x}/{y}.png'
+const onlineTileOptions = {
+        maxNativeZoom: 22,
+        maxZoom: 100,
+        subdomains: ['mt0','mt1','mt2','mt3'] 
+      }
+const offlineTileOptions = {
+        maxNativeZoom: 22,
+        maxZoom: 100,
+      }
+
+export default {
+  name: 'AutonRoverMap',
+
+  components: {
+    LMap,
+    LTileLayer,
+    LMarker,
+    LPolyline,
+    LPopup,
+    LTooltip,
+    LControlScale
+  },
+
+  created: function () {
+    // Get Icons for Map
+    this.locationIcon = L.icon({
+      iconUrl: '/static/location_marker_icon.png',
+      iconSize: [40, 40],
+      iconAnchor: [20, 20]
+    })
+    this.tangentIcon = L.icon({
+      iconUrl: '/static/gps_tangent_icon.png',
+      iconSize: [44, 80],
+      iconAnchor: [22, 60]
+    })
+    this.targetBearingIcon = L.icon({
+      iconUrl: '/static/gps_tangent_icon.png',
+      iconSize: [30, 56],
+      iconAnchor: [15, 42]
+    })
+    this.waypointIcon = L.icon({
+      iconUrl: '/static/map_marker.png',
+      iconSize: [64, 64],
+      iconAnchor: [32, 64],
+      popupAnchor: [0, -32]
+    })
+    this.projectedPointIcon = L.icon({
+      iconUrl: '/static/map_marker_projected.png',
+      iconSize: [64, 64],
+      iconAnchor: [32, 64],
+      popupAnchor: [0, -32]
+    })
+    this.postIcon = L.icon({
+      iconUrl: '/static/gate_location.png',
+      iconSize: [64, 64],
+      iconAnchor: [32, 64],
+      popupAnchor: [0, -32]
+    })
+
+    // TODO: Remove these and replace with ROS topics if still needed
+    //
+    // this.$parent.subscribe('/projected_points', (msg) => {
+    //   let newProjectedList = msg.points
+    //   this.projectedPoints = newProjectedList.map((projected_point) => {
+    //     return {
+    //       latLng: L.latLng(
+    //         projected_point.latitude_deg + projected_point.latitude_min/60,
+    //         projected_point.longitude_deg + projected_point.longitude_min/60
+    //       )
+    //     }
+    //   })
+
+    //   this.projectedPointsType = msg.path_type
+    // })
+
+    // this.$parent.subscribe('/estimated_gate_location', (msg) => {
+    //   this.post1 =  L.latLng(
+    //     msg.post1.latitude_deg + msg.post1.latitude_min/60,
+    //     msg.post1.longitude_deg + msg.post1.longitude_min/60
+    //   )
+    //   this.post2 = L.latLng(
+    //     msg.post2.latitude_deg + msg.post2.latitude_min/60,
+    //     msg.post2.longitude_deg + msg.post2.longitude_min/60
```

Comment:
I didn't really think much of it before but we should probably just rid of this stuff before merging to main. There's no reason if someone wanted to reference this in the future they couldn't just go back and look at the code in mrover-workspace